pageserver/src/layered_repository/layer_map.rs

```diff
@@ -183,6 +183,14 @@ impl LayerMap {
         }
     }
 
+    pub fn layer_exists_at_lsn(&self, seg: SegmentTag, lsn: Lsn) -> bool {
```

Comment:
This needs a comment to explain what this does. How is this different from the `exists_at_lsn` function?

Reply:
This is just a public wrapper, like `newer_image_layer_exists()` function